examples/test/test_ai_society_example.py

```diff
@@ -0,0 +1,26 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+import pytest
+
+import examples.ai_society.role_playing
+from camel.typing import ModelType
+
+
+@pytest.mark.slow
```

Comment:
When will this test stop? There is a for loop in the example. Should we put `"CAMEL_TASK_DONE"` in the output string of `STUB`?

Reply:
The for loop is bound to 50 iterations. So no need for `"CAMEL_TASK_DONE"`